### DISCUSSION TITLE
Failing on pre-commit

### DISCUSSION THREAD

**Discussion by darduf** [snok/drf-openapi-tester#273] [Q&A]
Hi there,

I have followed the contribution guidelines and I want to commit and push a branch to my fork of drf-openapi-tester to create a PR.

However the pre-commit checks are failing on `flake8` and `pylint` on lines of code not related to mine 

flake8
```
openapi_tester/loaders.py:166:49: TC006 Annotation 'APIView' in typing.cast() should be a string literal
openapi_tester/loaders.py:193:21: TC006 Annotation 'dict' in typing.cast() should be a string literal
openapi_tester/loaders.py:217:21: TC006 Annotation 'dict' in typing.cast() should be a string literal
openapi_tester/loaders.py:248:17: TC006 Annotation 'dict' in typing.cast() should be a string literal
```
pylint
```
openapi_tester/loaders.py:123:0: R0022: Useless option value for 'disable', 'no-self-use' was moved to an optional extension, see https://pylint.pycqa.org/en/latest/whatsnew/2/2.14/summary.html#removed-checkers. (useless-option-value)
```

What should I do here in order to get my PR in?

**Comment by sondrelg** [comment]
The TC006 errors can be fixed by changing this type of code

```python
cast(int, my_variable)
```
to 
```python
cast("int", my_variable)
```

And I'm not sure about the pylint error, but feel free to submit a PR with this unsolved and I can fix it later if needed 🙂

**Reply by darduf** [reply]
Thanks - so I will include the TC006 error fix in my fix-based PR?

How can I bypass the pylint error in my commit in order to submit the PR?

**Reply by sondrelg** [reply]
`git commit -m "your message" --no-verify` will let you commit 🙂
